packages/styleguide/src/components/Note/Note.js

```diff
@@ -5,42 +5,45 @@ import cx from 'classnames';
 
 const propTypes = {
   title: string,
-  /* type influences set styles */
-  type: oneOf(['warning', 'success', 'error'])
+  /* variant influences set styles */
+  variant: oneOf(['warning', 'success', 'error', 'info'])
```

Comment:
pekne pomenovanie ;)

Reply:
Leeeen teraz sa z toho stava breaking change - existujuce Note v projektoch prestanu spavne funguvat. Cize:
* bud si to nechame variants a otvorime si nejaku novu branchu, napr. `styleguide-next` a tam toto mergneme
* alebo to vratime na type a pridame info a v nejakom dalsiom PR si to premenujeme na `variant` a to budeme mergneme do `styleguide-next` napr.

Reply:
nechame type a variant mergneme inokedy. nemyslím si že nám treba túto breaking change.